src/EditorFeatures/VisualBasic/EndConstructGeneration/EndConstructStatementVisitor.vb

```diff
@@ -51,6 +51,27 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.EndConstructGeneration
             End If
         End Function
 
+        Public Overrides Function VisitExpressionStatement(node As ExpressionStatementSyntax) As AbstractEndConstructResult
+            Dim invocationExpression = TryCast(node.Expression, InvocationExpressionSyntax)
+            If invocationExpression IsNot Nothing AndAlso
+                    invocationExpression.ArgumentList IsNot Nothing AndAlso
+                    invocationExpression.ArgumentList.OpenParenToken.IsMissing AndAlso
+                    invocationExpression.ArgumentList.CloseParenToken.IsMissing Then
+                Dim newArgumentList = invocationExpression.ArgumentList _
+                    .WithOpenParenToken(SyntaxFactory.Token(SyntaxKind.OpenParenToken, "(")) _
+                    .WithCloseParenToken(SyntaxFactory.Token(SyntaxKind.CloseParenToken, ")"))
```

Comment:
I suspect this will blow away trailing trivia. Consider the case of Foo Arg1, Arg2 ' Comment